src/test-helpers.tsx

```diff
@@ -967,6 +971,10 @@ export const createFakeCommentsResponse = (
   return createFakeApiPage<GetCommentsResponse>({ results });
 };
 
+export const createErrorResponse = (errorResponse: { error: Error }) => {
+  return errorResponse as ErrorResponseType;
```

Comment:
We try to avoid type assertions because they bypass some type checking. In this case, it works just the same to declare a return type of the function so I suggest doing that instead:

```diff
diff --git a/src/test-helpers.tsx b/src/test-helpers.tsx
index 17ce432..9da3169 100644
--- a/src/test-helpers.tsx
+++ b/src/test-helpers.tsx
@@ -971,8 +971,10 @@ export const createFakeCommentsResponse = (
   return createFakeApiPage<GetCommentsResponse>({ results });
 };
 
-export const createErrorResponse = (errorResponse: { error: Error }) => {
-  return errorResponse as ErrorResponseType;
+export const createErrorResponse = (errorResponse: {
+  error: Error;
+}): ErrorResponseType => {
+  return errorResponse;
 };
 
 export const setMockFetchResponseJSON = (
```

Reply:
Fixed, declared strong type checking for the return type.